client/.env.example

```diff
@@ -1,2 +1,3 @@
 REACT_APP_BASE_URL=port
-RAZOR_PAY_KEY =RAZOR_PAY_KEY
\ No newline at end of file
+RAZOR_PAY_KEY =RAZOR_PAY_KEY
+CHECKOUT=https://checkout.razorpay.com/v1/checkout.js
```

Comment:
Add REACT_APP_ before RAZOR_PAY_KEY and CHECKOUT

Reply:
@Gangotri-sg3 have you added this?

Reply:
removed it